test/content.write.js

```diff
@@ -267,15 +289,16 @@ test('checks the size of stream data if opts.size provided', (t) => {
   })
 })
 
-test('only one algorithm for now', t => {
-  t.throws(() => write(CACHE, 'foo', { algorithms: [1, 2] }), {
+t.test('only one algorithm for now', async t => {
+  const CACHE = t.testdir()
+  await t.rejects(() => write(CACHE, 'foo', { algorithms: [1, 2] }), {
```

Comment:
this _technically_ is a breaking change as we're changing from a synchronously thrown error to a promise rejection, however given that all of the other errors in this method were already promise rejections i think we can call it a fix instead

Reply:
This is 100% a fix, folks would already have been having to `.catch()` because of other promise rejections potentially done here.